pyproject.toml

```diff
@@ -23,15 +23,15 @@ classifiers = [
 ]
 requires-python = '>=3.9,<3.13'
 dependencies = [
-  'asteval',       # For evaluating mathematical expressions
-  'easyscience',   # The base library of the EasyScience framework
-  'gemmi',         # For handling CIF files
-  'numpy<2.0',     # For numerical calculations
-  'periodictable', # For atomic properties
-  'pint',          # For units
-  'uncertainties', # For error propagation
-  'versioningit',  # For versioning
-  'xarray',        # For handling data arrays
+  'asteval',                                                                     # For evaluating mathematical expressions
+  'easyscience @ git+https://github.com/EasyScience/EasyScience.git@DescriptorAnyType', # The base library of the EasyScience framework
+  'gemmi',                                                                       # For handling CIF files
+  'numpy',                                                                   # For numerical calculations
+  'periodictable',                                                               # For atomic properties
+  'pint',                                                                        # For units
+  'uncertainties',                                                               # For error propagation
+  'versioningit',                                                                # For versioning
+  'xarray',                                                                      # For handling data arrays
```

Comment:
this formatting will fail the linter. Please run `npx` as described in 
https://github.com/EasyScience/EasyDiffractionLib/blob/master/DEVELOPMENT.md